apps/src/imageUtils.js

```diff
@@ -105,38 +96,71 @@ export async function dataURIToBlob(uri) {
  * "data:image/png;base64,iVBOR..."
  */
 
+/**
+ * Given an input of a supported type, converts it to an HTMLImageElement.
+ *
+ * @param {Blob|HTMLImageElement|ImageURI} input
+ * @returns {Promise<HTMLImageElement>}
+ */
+export async function toImage(input) {
+  if (input instanceof HTMLImageElement) {
+    return input;
+  }
+
+  let src;
+  let cleanup = () => {};
+
+  if (input instanceof Blob) {
+    src = URL.createObjectURL(input);
+    cleanup = () => URL.revokeObjectURL(input);
```

Comment:
For more on these methods, see ["Using object URLs to display images"](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications#Example_Using_object_URLs_to_display_images) and ["Using URL.createObjectURL()"](https://chrisrng.svbtle.com/using-url-createobjecturl).